common/src/main/java/com/wynntils/core/components/Handler.java

```diff
@@ -9,7 +9,7 @@
  * aspect of Minecraft functionality, and with some Wynncraft knowledge, they distribute
  * the MC content to the actual Wynncraft models. A handler is needed when there is not a
  * clear 1-to-1 relationship between Minecraft components and models.
- *
+ * <p>
```

Comment:
Revert this. We are not generating html, but reading javadoc in the source. This just hurts readability.

Reply:
... and all the other changes of the same kind.